googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveConsumer.java

```diff
@@ -95,7 +95,7 @@ protected int poll() throws Exception {
                 postProcess(fileModel, file);
             }
         } catch (GoogleJsonResponseException e) {
-            LOG.error("Google Drive Consume error:", e);
+            LOG.info(" >> GoogleJsonResponseException occurred");
             service = this.endpoint.getClient(true);
```

Comment:
Not good to set an exception in an info log. Better to change it into a warning. Also, when you get this as user, you want to have more info. I would suggest something like this (pseudocode):

if(connectionAttempts < 60){
  LOG.warn("Could not connect to Google Drive directory + directory. Try again after x seconds (attempt " + connectionAttempts + " of 60)");
  connectionAttempts += 1;
  service = this.endpoint.getClient(true);  
}else{
  LOG.error("GoogleJsonResponseException occurred. Connect to Google Drive directory + directory failed:", e);
}

Reply:
x = the poll interval. The user can change the poll interval to try it for a longer period.

Reply:
changes done!
x = should be the delay parameter set by the user